app/frontend/src/features/auth/password/ChangePasswordPage.tsx

```diff
@@ -82,8 +82,10 @@ export default function ChangePasswordPage() {
             </Alert>
           )}
           <Typography className={classes.infoText} variant="body1">
-            Please enter a new password, or leave the "New password" and
-            "Confirm password" fields blank to unset your password.
+            Please enter a new password,{" "}
+            {accountInfo?.hasPassword
+              ? 'or leave the "New password" and "Confirm password" fields blank to unset your password.'
+              : "or leave the fields blank to unset your password."}
```

Comment:
Still the same problem, no? The user has no password and they see `Please enter a new password, or leave the fields blank to unset your password.` But they already don't have a password.

Reply:
Oh yeah what was I thinking 🤦